<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Sandy" failures="2" tests="2" name="Add Customer Test" time="1.971" errors="0" timestamp="28 Feb 2022 16:09:09 GMT">
  <testcase classname="com.w2a.testcases.AddCustomerTest" name="addCustomerTest" time="0.589">
    <failure type="java.lang.AssertionError" message="Customer not added successfully">
      <![CDATA[java.lang.AssertionError: Customer not added successfully
at com.w2a.testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:40)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
  <testcase classname="com.w2a.testcases.AddCustomerTest" name="addCustomerTest" time="0.428">
    <failure type="java.lang.AssertionError" message="Customer not added successfully">
      <![CDATA[java.lang.AssertionError: Customer not added successfully
at com.w2a.testcases.AddCustomerTest.addCustomerTest(AddCustomerTest.java:40)
... Removed 26 stack frames]]>
    </failure>
  </testcase> <!-- addCustomerTest -->
</testsuite> <!-- Add Customer Test -->
